### ISSUE TITLE
Semantic highlighting/Intellisense not working after VS 15.5 reload with new SDK

### ISSUE THREAD

**Issue by forki** [Microsoft/visualfsharp#3739]
#### Repro steps

1. git clone https://github.com/SAFE-Stack/SAFE-BookStore.git
1. cd SAFE-BookStore
1. open BookStore.sln
1. open `src/Server/Program.fs`
1. restore runs in background automatically
1. wait until Semantic Highlighting is there
1. build sln (Ctrl+Shift+b)
1. close sln
1. reopen sln
1. open `src/Server/Program.fs`

#### Expected behavior

Semantic highlighting/Intellisense working

#### Actual behavior

Semantic highlighting/Intellisense not working

#### Known workarounds

* close sln
* Go to task manager and kill all MSBuild processes

#### Related information

Looks related to https://github.com/Microsoft/visualfsharp/issues/3222 but is with new sdk projects and new project system

// cc @vasily-kirichenko @0x53A @davkean

**Comment by forki** [comment]
ok so this is actually exact duplicate of #3222 but with the new sdk.
When I delete *CoreCompileInputs.cache after step 8 then F# feature work fine.
I'm closing this in favor of #3222

**Comment by forki** [comment]
But just to make that point very clear: all new-style fsproj projects are now broken after first build and reopening of sln

**Comment by forki** [comment]
ok one new workaround: 

After step 9: 
* Run "BUILD/Clean Solution"
* Run "BUILD/Build Solution"

==> F# features come back

BUT:

if you just run "BUILD/Rebuild solution" they don't come back.

Is it that Rebuild doesn't call Clean properly?

**Comment by forki** [comment]
Reopening since @kevinransom fixed #3222 with a change to old project system. So this is probably still broken.

**Comment by cartermp** [comment]
@forki Have you confirmed this on 15.6 Preview 6? The issue in #3222 comes from an issue with referencing the FCT package in the old project system where a fix that @KevinRansom made wasn't available.

I noticed a similar symptom here: https://github.com/Microsoft/visualfsharp/issues/3222#issuecomment-351475793, but I can no longer reproduce it on the latest 15.6 preview.

**Comment by forki** [comment]
No I did not and I won't have time in the next few days.  But repro is
there....

Am 28.02.2018 17:23 schrieb "Phillip Carter" <notifications@github.com>:

**Comment by cartermp** [comment]
This was fixed in 15.6.
